apps/admin-gui/src/app/vos/pages/vo-detail-page/vo-overview/vo-overview.component.html

```diff
@@ -1,8 +1,9 @@
 <mat-spinner *ngIf="loading" class="mr-auto ml-auto"></mat-spinner>
 <div *ngIf="!loading">
+  <app-statistics-card
+    [rowNames]="rowNames"
+    [statistics]="membersCount"
+    [title]="'VO_DETAIL.OVERVIEW.MEMBERS_COUNT'">
+  </app-statistics-card>
```

Comment:
I suggest moving the statisticts under the perun-web-apps-menu-buttons-field
I dont think that the statistics are that important so they are at the top, the box button are more important in my opinion